bundles/org.openhab.binding.folderwatcher/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="folderwatcher"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="ftpfolder">
+		<label>FTP Folder</label>
+		<description>FTP folder to be watched</description>
+
+		<channels>
+			<channel id="newfile" typeId="newfile-channel" />
+		</channels>
+
+		<config-description>
+			<parameter name="ftpAddress" type="text" required="true">
+				<label>FTP Server</label>
+				<description>Address of FTP server</description>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="ftpPort" type="integer" min="1" max="65535">
```

Comment:
I see you already have the limits set. The framework makes sure that these limits are observed.